### DISCUSSION TITLE
发现pop 组件的bug,指定内部pop-content 下的内容可以滑动，无法生效，后续我在该类名下外层使用了 list组件可以滑动，但是只能向下滑动，不能向上滑动。

### DISCUSSION THREAD

**Discussion by L1-corona** [hellof2e/quark-design#318] [Q&A]
![a33c08d9b9ec2d1039f22418cece094](https://github.com/hellof2e/quark-design/assets/80499682/d1b0cbae-0af4-4ba2-af4e-600296ecde5a)
![dd7dd572e18725f2767233b6cae1554](https://github.com/hellof2e/quark-design/assets/80499682/d6bf3e94-f13d-4418-b977-ec4492dc50a5)

以上是我代码截图，由于该组件被我使用在公司项目，发现了不少棘手的bug,后续换成京东nut UI后解决

**Comment by sanqi-med** [comment]
我们看下
